src/plugins/persistence/couch/CouchSearchProvider.js

```diff
@@ -32,7 +32,7 @@ class CouchSearchProvider {
         this.couchObjectProvider = couchObjectProvider;
     }
 
-    search(query, abortSignal) {
+    searchForObjects(query, abortSignal) {
```

Comment:
if these queries ever prove to be slow, we can look at switching to couch "views" instead